src/Math-Statistics/PMStatisticalSample.class.st

```diff
@@ -49,6 +49,20 @@ PMStatisticalSample class >> harmonicMean: aCollection [
 	^ (self newFrom: aCollection) harmonicMean
 ]
 
+{ #category : #information }
+PMStatisticalSample class >> mean: aCollection [
```

Comment:
maybe `aCollection` could be `aSample`

Reply:
ok, I changed from aCollection to aSample